NEW_MODULE_TECH_EVAL.MD

```diff
@@ -81,6 +81,7 @@ Information gathered includes:
     * When the Technical council's decision is contrary to the evaluation, or after any provisional acceptance or deferral, the Technical Council is required to provide written justification for their decision.
 1. Upon completion of the review, the JIRA will be updated to indicate this.
     * JIRA workflow: the ticket is transitioned from **TC REVIEW** to **APROVED** or **REJECTED**
+1. The Technical Council will notify the Release Management Stakeholders Group that the module has been approved.
```

Comment:
Deliberately not saying "approved for X flower release", since currently the flower releases are not part of our process.  Discussion at the TC meeting that we might change this in the future.  But for now RSMG can interpret the approval and make their own connection to the flower releases based on when it was approved.